client/asset/doge/doge.go

```diff
@@ -189,3 +215,44 @@ func NewWallet(cfg *asset.WalletConfig, logger dex.Logger, network dex.Network)
 
 	return btc.BTCCloneWallet(cloneCFG)
 }
+
+// externalFeeEstimator gets the fee rate from the external API and returns it
+// in sats/vByte.
+func externalFeeEstimator(ctx context.Context, net dex.Network) (uint64, error) {
+	var url string
+	if net == dex.Testnet {
+		url = testnetExternalApiUrl
+	} else {
+		url = externalApiUrl
+	}
+	url = url + "fee/1"
+	ctx, cancel := context.WithTimeout(ctx, 4*time.Second)
+	defer cancel()
+	r, err := http.NewRequestWithContext(ctx, http.MethodGet, url, nil)
+	if err != nil {
+		return 0, err
+	}
+	httpResponse, err := http.DefaultClient.Do(r)
+	if err != nil {
+		return 0, err
+	}
+	var resp map[string]float64
+	reader := io.LimitReader(httpResponse.Body, 1<<20)
+	err = json.NewDecoder(reader).Decode(&resp)
+	if err != nil {
+		return 0, err
+	}
+	httpResponse.Body.Close()
+
+	feeBtcByte, ok := resp["feerate"]
+	feeInSat := toSatoshi(feeBtcByte / 1000)
```

Comment:
Are you sure the units are `feeBtcByte`?  I think the bitcore result uses exactly the same as the units from `estimatesmartfee`.  DOGE/kB?

Reply:
You are correct, I didn't pay attention to the `DefaultFeeRateLimit` comment and thought 6 doge/kb was too high to be right. Fixed it